docs/openapi/schemas.yaml

```diff
@@ -1517,3 +1520,91 @@ ImportAssistantResponse:
     choices:
       - message:
           content: '.breadcrumbs'
+ApiKeyRequest:
+  type: object
+  required:
+    - name
+    - feature
+    - domains
+  properties:
+    name:
+      description: The name of the API key
+      type: string
+    feature:
+      description: The feature the API key is for
+      type: string
+    domains:
+      description: The domains the API key is allowed to access
+      type: array
+      items:
+        type: string
+  example:
+    name: 'Example API Key'
+    feature: 'imports'
+    domains:
+      - 'example.com'
+ApiKeyResponse:
+  type: object
+  properties:
+    id:
+      description: The ID of the API key
+      $ref: '#/Id'
+    name:
+      description: The name of the API key
+      type: string
+    apiKey:
+      description: The API key
+      type: string
+    scopes:
+      description: The scopes of the API key
+      type: array
+      items:
+        type: string
+    domains:
+      description: The domains the API key is allowed to access
+      type: array
+      items:
+        type: string
+    imsOrgId:
+      description: The IMS organization ID
+      $ref: '#/ImsOrganizationId'
+    imsUserId:
+      description: The IMS user ID
+      $ref: '#/ImsUserId'
+    createdAt:
+      description: The date and time when the API key was created
+      $ref: '#/DateTime'
+    expiresAt:
+      description: The date and time when the API key expires
+      $ref: '#/DateTime'
```

Comment:
Do we need a `deletedAt` here as well, to handle when the user initiates a delete?

Reply:
Yes - can add a deletedAt, should work.

Reply:
Should deletedAt be part of the ApiKeyMetadataResponse? As `ApiKeyResponse` is shown only once - when the user creates an API Key.

Reply:
We list the expired key as part of the APIKeyMetadataResponse, but not the deleted keys.

Reply:
`deletedAt` will be an attribute in the ApiKeys table. We do not need to show it in the metadata response.